NAMESPACE

```diff
@@ -411,6 +419,18 @@ export(scale_fill_grey)
 export(scale_fill_hue)
 export(scale_fill_identity)
 export(scale_fill_manual)
+export(scale_fill_off_continuous)
+export(scale_fill_off_date)
+export(scale_fill_off_datetime)
+export(scale_fill_off_discrete)
+export(scale_fill_off_gradient)
+export(scale_fill_off_gradient2)
```

Comment:
@csaluski I don't think this is correct... there should not be a separate scale for color_off,fill_off. The color_off and color values should be combined into a single scale.